quickstart/dev/appcheck/README.md

```diff
@@ -0,0 +1,140 @@
+## Device Login Flow for App Check
```

Comment:
I don't think this file is really necessary? maybe it could be moved outside of `/dev/`

Reply:
Do you mean deleted from the main repo https://github.com/GaloyMoney/galoy/tree/main/dev/appcheck? Or just not sync it into [quickstart](https://github.com/GaloyMoney/galoy/blob/9bf8a3be989d15897c54a59727732b59eab7f89c/quickstart/vendir/template.yml#L14)?